src/app/assets/scss/components/_article-card.scss

```diff
@@ -140,14 +140,14 @@ svg {
 }
 
 .read-more {
+  background: $ui-02;
   background: $ui-02;
   bottom: 6px;
+  bottom: 0;
```

Comment:
also duplicated here? need to remove the `bottom: 6px` above it?

Reply:
looks like some of eric's changes intended 6px. I removed the duplicate. probably could have been avoided if style lint is run consistently on check in or we put a pre-commit check.